### PR TITLE
new SpaceList component

### PR DESCRIPTION
**Pull request by TyaglovOV**
Implement space avatar icon, space avatar icon animation (svg), space list with space avatar selecting, create space button (without modal window), scrolling.

### CODE REVIEW

File: src/components/SpaceList/SpaceList.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 dialog LLC <info@dlg.im>
+ * @flow
+ */
+
+import type { AvatarPlaceholder } from '@dlghq/dialog-types';
+
```
Comment (meafmira) [inline src/components/SpaceList/SpaceList.js]: Use empty line only for separate absolute imports from  relative

File: src/components/SpaceList/SpaceAdd.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    return (
+      <div
+        className={className}
+        style={{ minWidth: size, width: size, height: size }}
+      >
+        <svg
+          viewBox="0 0 100 100"
+          shapeRendering="auto"
+        >
+          <g onClick={this.handleClick}>
```
Comment (meafmira) [inline src/components/SpaceList/SpaceAdd.js]: Simply use `this.props.onClick`

**Comment by meafmira** [comment]
@TyaglovOV sign (verify) your commits, pls (https://help.github.com/articles/signing-commits/)

File: src/components/SpaceList/SpaceAdd.js (hunk 2 above)
Comment (olegshilov) [reply src/components/SpaceList/SpaceAdd.js]: maybe we should handle click on container not on svg group

File: src/components/SpaceList/SpaceList.js (hunk 1 above)
Comment (TyaglovOV) [reply src/components/SpaceList/SpaceList.js]: done

**Comment by TyaglovOV** [comment]
> @TyaglovOV sign (verify) your commits, pls (https://help.github.com/articles/signing-commits/)

+

**Comment by meafmira** [comment]
> > @TyaglovOV sign (verify) your commits, pls ([help.github.com/articles/signing-commits](https://help.github.com/articles/signing-commits/))
> 
> done

No. You didn't sign your previously pushed commits. We can't merge branch with unsigned commits

**Comment by TyaglovOV** [comment]
> > > @TyaglovOV sign (verify) your commits, pls ([help.github.com/articles/signing-commits](https://help.github.com/articles/signing-commits/))
> > 
> > 
> > done
> 
> No. You didn't sign your previously pushed commits. We can't merge branch with unsigned commits

done

**Pull request merged**